src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/GoToImplementationTests.cs

```diff
@@ -16,8 +16,8 @@ public async Task GoToImplementation_SameFile()
 
             await TestServices.Editor.PlaceCaretAsync("IncrementCount", charsOffset: -1, ControlledHangMitigatingCancellationToken);
 
-            // Act
-            await TestServices.Editor.InvokeGoToImplementationAsync(ControlledHangMitigatingCancellationToken);
+            // Act (Ctrl+12 == GoToImplementation
+            TestServices.Input.Send("^{F12}");
```

Comment:
Why this change? Seems flakier, and also worrying if invoking commands doesn't work. Not everything has a shortcut key